core/settings.py

```diff
@@ -0,0 +1,127 @@
+"""
+Django settings for core project.
+
+Generated by 'django-admin startproject' using Django 5.1.4.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/5.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/5.1/ref/settings/
+"""
+import os
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/5.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = os.environ.get("SECRET_KEY","django-insecure-tp&ct@$wkoi0_tcn^=#!=rkd(yj48w=xs_%t489^%p_&oczjdl")
```

Comment:
Ensure that the `SECRET_KEY` environment variable is set in production environments to avoid using the insecure default value.